packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 3.7.0 - Apr 06, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- Provide multiple entry points of creating testing tenant/accounts to unblock user from M365 account issues, like M365 account does not have sideloading permission or user does not have M365 account at all. These entry points include an Add(+) button besides ACCOUNTS in sidebar, an new "create account" option in `Teams: Account` Command and improved Get Started page.
+
+Enhancement:
```

Comment:
@zyxiaoyuer Maybe this UX change is an improvement? https://github.com/OfficeDev/TeamsFx/pull/4402

Reply:
@tecton This is referring to the TTK integration with ITP portal, should be a new feature. This feature provides entry points for users to create ITP account easily at different stage, e.g. get started/login/encounter errors when F5. It contains more than PR#4402.

Reply:
OK, I just want to check that whether the improvement should be included. They are different scopes.